ckanext/publicamundi/lib/actions/package.py

```diff
@@ -60,6 +61,40 @@ def dataset_export(context, data_dict):
     
     return result
 
+@logic.side_effect_free
+def dcat_export(context, data_dict):
+    '''Export a dataset to RDF XML using GeoDCAT XSLT.
+
+    :param id: the name or id of the dataset to be exported.
+    :type id: string
+
+    rtype: string
+    '''
+
+    pkg = _get_action('package_show')(context, data_dict)
+    dtype = pkg.get('dataset_type')
+    obj = pkg.get(dtype) if dtype else None
+    result = None
+    if obj:
+        # Get a proper serializer
+        xser = xml_serializer_for(obj)
+        xser.target_namespace = config.get('ckan.site_url')
+        # Get the XML
+        tmp_xml = xser.dumps()
```

Comment:
Fixed in https://github.com/kalxas/ckanext-publicamundi/commit/ceb196178a05aad4348565214c55fb5dd7b91336